src/ParseServiceProvider.php

```diff
@@ -70,6 +70,8 @@ protected function setupParse()
 
             ParseClient::setServerURL($serverURL, $mountPath);
         }
+
+        ParseClient::setStorage(new ParseSessionStorage());
```

Comment:
Sorry, since this is my first time submitting a PR, please bear with me. Are you suggesting that we should add this setting inside the config file then check whether the user wanna use Laravel Session or the global $_SESSION?